config/samples/webhook_v1alpha1_imagemap.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: webhook.image-relocation.pivotal.io/v1alpha1
+kind: ImageMap
+metadata:
+  name: bootcamp-sample
+spec:
+  map:
+    "gcr.io/google-samples/kubernetes-bootcamp:v1": "<repo prefix>/kubernetes-bootcamp:v1"
```

Comment:
It's generally advisable to avoid unbounded keys, except for the most simple map[string]string.

While this is a map of strings, it may still be better to decompose it into an array of structs, like:

```yams
- origin: gcr.io/google-samples/kubernetes-bootcamp:v1
  destination: <repo prefix>/kubernetes-bootcamp:v1
- ...
```

Reply:
Done in https://github.com/pivotal/kubernetes-image-mapper/pull/2